app/Http/Controllers/UserController.php

```diff
@@ -91,28 +91,26 @@ public function performSignup(Request $request) {
         }
         else {
             // email activation is necessary
+        	$active = 0;
             $response = redirect(route('login'))->with('success', 'Thanks for signing up! Please confirm your email to continue..');
-            $active = 0;
         }
 
-        $api_active = false;
-        $api_key = null;
-
         if (env('SETTING_AUTO_API')) {
             // if automatic API key assignment is on
-            $api_active = 1;
+            $api_active = TRUE;
```

Comment:
Could you use `false`, `null` rather than `FALSE`, `NULL` to be consistent with the rest of the code?